packages/vaadin-virtual-list/src/virtualizer-iron-list-adapter.js

```diff
@@ -61,7 +61,7 @@ export class IronListAdapter {
   }
 
   scrollToIndex(index) {
-    if (typeof index !== 'number' || isNaN(index) || !this.scrollTarget.offsetHeight) {
+    if (typeof index !== 'number' || isNaN(index) || this.size === 0 || !this.scrollTarget.offsetHeight) {
```

Comment:
Skip scroll to index altogether if the virtualizer doesn't have any items. Scrolling to index with an empty virtualizer resulted in division by zero [here](https://github.com/vaadin/web-components/pull/1971/files#diff-5f471994c5bb4ba62dbd8dd477ecd39a5a9d6b7c771c7ddb7f42dc9ec140422aL70). This didn't cause a bug since the `_vidxOffset`, which resulted to `NaN`, was fixed back to 0 [here](https://github.com/vaadin/web-components/pull/1971/files#diff-5f471994c5bb4ba62dbd8dd477ecd39a5a9d6b7c771c7ddb7f42dc9ec140422aL406)